app/docs/dev/policies/meshaccesslog.md

```diff
@@ -0,0 +1,427 @@
+                  - key: "start_time"
+                    value: "%START_TIME%"
+          - file:
+              path: /tmp/access.log
+              format:
+                plain: '[%START_TIME%]'
+```
+
+Apply the configuration with `kubectl apply -f [..]`.
+
+{% endtab %}
+{% tab meshaccesslog-multiple-backends Universal %}
+
+```yaml
+type: MeshAccessLog
+name: default
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              address: 127.0.0.1:5000
+              format:
+                json:
+                  - key: "start_time"
+                    value: "%START_TIME%"
+          - file:
+              path: /tmp/access.log
+              format:
+                plain: '[%START_TIME%]'
+```
+
+Apply the configuration with `kumactl apply -f [..]` or with the [HTTP API](../../reference/http-api).
+
+{% endtab %}
+{% endtabs %}
+
+## Log all incoming and outgoing traffic
+
+{% tabs meshaccesslog-all useUrlFragment=false %}
+{% tab meshaccesslog-all Kubernetes %}
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshAccessLog
+metadata:
+  name: default
+  namespace: kuma-system
+  labels:
+    kuma.io/mesh: default # optional, defaults to `default` if unset
+spec:
+  targetRef:
+    kind: Mesh
+  from: # delete this section if you do not want to log incoming traffic
+    - targetRef:
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              path: /tmp/access.log
+  to: # delete this section if you do not want to log outgoing traffic
+    - targetRef:
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              path: /tmp/access.log
+```
+
+Apply the configuration with `kubectl apply -f [..]`.
+
+{% endtab %}
+{% tab meshaccesslog-all Universal %}
+
+```yaml
+type: MeshAccessLog
+name: default
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  from: # delete this section if you do not want to log incoming traffic
+    - targetRef:
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              path: /tmp/access.log
+  to: # delete this section if you do not want to log outgoing traffic
+    - targetRef:
+        kind: Mesh
+      default:
+        backends:
+          - file:
+              path: /tmp/access.log
+```
+
+Apply the configuration with `kumactl apply -f [..]` or with the [HTTP API](../../reference/http-api).
+
+{% endtab %}
+{% endtabs %}
+
+## Logging traffic going outside the Mesh
+
+To target [External Service](/docs/{{ page.version }}/policies/external-services#usage) use `MeshService` as a `targetRef` target and match external service `name`.
+
+To target traffic going through [passthrough mode](/docs/{{ page.version }}/policies/external-services) use `Mesh` as a `targerRef`. This will cause case `%KUMA_DESTINATION_SERVICE%` will have value `external` and `%UPSTREAM_HOST%` will have an IP of the service.
```

Comment:
```suggestion
To target [`ExternalServices`](/docs/{{ page.version }}/policies/external-services#usage), use `MeshService` as the `targetRef` kind with `name` set to  
the `kuma.io/service` value.

To target other non-mesh traffic, i.e. [passthrough traffic](/docs/{{ page.version }}/policies/mesh/#controlling-the-passthrough-mode), use `Mesh` as the `targetRef` kind. In this case, `%KUMA_DESTINATION_SERVICE%` is set to `external` and `%UPSTREAM_HOST%` is set to IP of the service.
```

Reply:
I'll apply this but I don't know why `UPSTREAM_HOST` is no longer relevant here.

Reply:
I don't know why it was ever relevant, it will have the obvious value like every other Envoy command operator.